code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,236 @@
+	var/list/names = list() //List of names of the people it's eaten
+	var/consumes = 0 //How many people it has eaten
+	environment_smash = 1
+	speak = list("Help!","Help me!","Somebody help!","Get over here, quickly!")
+
+
+/mob/living/simple_animal/hostile/wendigo/CanAttack(var/atom/the_target)
+	var/list/target_prox = view(the_target, vision_range)
+	for(var/obj/machinery/space_heater/campfire/fire in target_prox)
+		var/dist = get_dist(the_target, fire)
+		if(dist < (fire.light_range*2))//Just sitting on the edge of the fire
+			return
+
+	return ..(the_target)
+
+/mob/living/simple_animal/hostile/wendigo/AttackingTarget()
+	if(!target)
+		return
+
+	if(ismob(target))
+		var/mob/living/mob_target = target
+		if(mob_target.isDead() && !istype(mob_target, /mob/dead/observer))
+			set waitfor = 0
+			visible_message("<span class = 'notice'>\The [src] starts to take a bite out of \the [target].</span>")
+			stop_automated_movement = 1
+			if(do_after(src, mob_target, 50, needhand = FALSE))
+				playsound(get_turf(src), 'sound/weapons/bite.ogg', 50, 1)
+				var/damage = rand(melee_damage_lower, melee_damage_upper)
+				mob_target.adjustBruteLoss(damage)
+				if(health < maxHealth)
+					health = min(maxHealth,(health+damage))
+
+				if(ishuman(mob_target))
+					if(mob_target.health < -400)
+						visible_message("<span class = 'warning'>\The [src] is trying to eat \The [mob_target]!</span>","<span class = 'warning'>You hear crunching.</span>")
+						if(do_after(src, mob_target, 50, needhand = FALSE))
+							consumes += 1
+							names += mob_target.real_name
+							qdel(mob_target)
+
+			return
+	. =..()
+
+/mob/living/simple_animal/hostile/wendigo/Life()
+	/*Check evolve like zombies, run away from fire,
+	mimic speech if anyone's nearby but just out of eyesight
+	*/
+	..()
+	if(!isDead())
+		check_evolve()
+
+		var/list/can_see = view(src, vision_range)
+
+		for(var/obj/machinery/space_heater/campfire/fire in can_see)
+			var/dist = get_dist(src, fire)
+			if(dist < (fire.light_range*2))
+				walk_away(src,fire,(fire.light_range*2),move_to_delay)
+
+
+		if(names.len)
+			speak_chance = 15
+
+
+/mob/living/simple_animal/hostile/wendigo/GetVoice()
+	if(names.len)
+		return pick(names)
+	else
+		speak_chance = 0
+		return name
+/mob/living/simple_animal/hostile/wendigo/proc/check_evolve()
+	return
+
+/mob/living/simple_animal/hostile/wendigo/human
+	icon_state = "wendigo"
+	icon_living = "wendigo"
+	icon_dead = "wendigo_dead"
+
+	speed = 1
+
+/mob/living/simple_animal/hostile/wendigo/human/death()
+	flick("wendigo_dying",src)
+	..()
+
+/mob/living/simple_animal/hostile/wendigo/human/check_evolve()
+	if(consumes > HUMEVOLV)
+		var/mob/living/simple_animal/hostile/wendigo/evolved/new_wendigo = new /mob/living/simple_animal/hostile/wendigo/evolved(src.loc)
+		new_wendigo.names = names
+		qdel(src)
+
+/mob/living/simple_animal/hostile/wendigo/evolved
+	icon = 'icons/mob/48x48.dmi'
```

Comment:
needs a name = "evolved wendigo".  it looks janky that they're all named the same.